package.json

```diff
@@ -26,17 +26,19 @@
     "@angular/platform-browser-dynamic": "^12.2.0",
     "@angular/router": "^12.2.0",
     "@nestjs/cli": "^8.1.4",
-    "@nestjs/common": "^7.0.0",
-    "@nestjs/config": "^1.0.2",
-    "@nestjs/core": "^7.0.0",
-    "@nestjs/platform-express": "^7.0.0",
+    "@nestjs/common": "^8.2.0",
```

Comment:
This seems to break the unit tests

Reply:
jo @nestjs/testing war noch auf version 7